src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -597,6 +609,11 @@ export default class Blockchain extends Emittery<BlockchainTypedEvents> {
   ) => {
     const nextBlock = this.#readyNextBlock(this.blocks.latest, timestamp);
 
+    // if block time is incremental, adjustments should only apply once,
+    // otherwise they accumulate with each block.
+    if (this.#options.miner.timestampIncrement !== "clock") {
+      this.#timeAdjustment = 0;
+    }
```

Comment:
```suggestion
```
This is duplicating code below and can be removed. Seems like a rebase artifact.